pkg/imageworker/worker.go

```diff
@@ -62,15 +62,15 @@ func (w *Worker) Start(workers uint, stop <-chan struct{}) error {
 	}()
 
 	for i := uint(0); i < workers; i++ {
```

Comment:
could simplify this a bit to 
```suggestion
	for range workers {
```
(Go 1.22 ftw 🍾 )